src/app/admin/banner/index.tsx

```
@@ -0,0 +1,79 @@
+      </Head>
+      <div className="mb-14 flex flex-row items-end justify-between">
+        <Heading>배너 관리</Heading>
+        <div
+          onClick={() =>
+            handleModal({
+              title: '배너 업로드',
+              content: <CreateBanner closeModal={closeModal} />,
+            })
+          }
+        >
+          <div className="-mr-3 p-1 opacity-40 transition-opacity hover:opacity-70 sm:hidden">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)